src/main/kotlin/com/github/julianhille/sqldelight/jscipherdriver/JsSqlCipherDriver.kt

```diff
@@ -0,0 +1,207 @@
+package com.github.julianhille.sqldelight.drivers.jscipher
+
+import com.squareup.sqldelight.db.SqlCursor
+import com.squareup.sqldelight.db.SqlDriver
+import com.squareup.sqldelight.db.SqlPreparedStatement
+import com.squareup.sqldelight.Transacter
+import org.khronos.webgl.Uint8Array
+import org.khronos.webgl.Int8Array
+
+
+@JsNonModule @JsModule("better-sqlite3-multiple-ciphers")
+external fun createDatabase(path: String? = definedExternally, options: DatabaseOptions? = definedExternally): Database
+
+class DatabaseConfiguration(
+  val name: String,
+  var path: String,
+  var schema: SqlDriver.Schema,
+  var create: (SqlDriver) -> Unit,
+  var upgrade: (SqlDriver, Int, Int) -> Unit,
+  var journalMode: Boolean = true,
+  val memory: Boolean = false,
+  var key: String? = null,
+) {
+
+  fun dbPath(): String {
+    if (memory) return ":memory:"
+    return path + name
+  }
+}
+
+class SqlJsCipherDriver (var configuration: DatabaseConfiguration ): SqlDriver {
+  private val db = createDatabase(configuration.dbPath(), js("{ verbose: console.log }"))
+  private val statements = mutableMapOf<Int, SqlJsCipherStatement>()
+  private var transaction: Transacter.Transaction? = null
+
+
+  init {
+    if (configuration.journalMode ) db.pragma("journal_mode = WAL")
+    if (configuration.key?.isNotBlank() == true) db.pragma("journal_mode = WAL")
+    migrateIfNeeded(configuration.create, configuration.upgrade, configuration.schema.version)
+  }
+
+  override fun close() {
+    db.close()
+  }
+
+  override fun execute(identifier: Int?, sql: String, parameters: Int, binders: (SqlPreparedStatement.() -> Unit)?) {
+    createOrGetStatement(identifier, sql).run {
+      bind(binders)
```

Comment:
Not using `parameters` anywhere

`JsSqlDriver` does

```suggestion
      bind(parameters, binders)
```

Reply:
https://github.com/cashapp/sqldelight/blob/1.5.3/drivers/sqljs-driver/src/main/kotlin/com/squareup/sqldelight/drivers/sqljs/JsSqlDriver.kt#L40 you need to compare wikth 1.5.3 not with 2.x